p-token/src/processor/shared/initialize_account.rs

```diff
@@ -86,6 +89,10 @@ pub fn process_initialize_account(
         account.set_native(true);
         account.set_native_amount(minimum_balance);
         // SAFETY: single mutable borrow to `new_account_info` lamports.
+        // JC nit: up to you, but since you've already checked that
+        // `lamports() < minimum_balance` earlier, you don't actually need
+        // checked math. If it shaves CUs, go for it! But definitely add a
+        // comment explaining why
```

Comment:
Good point! Also we don't need to do the `borrow_lamports_unchecked `, since we are not updating its value.
```rust
// `new_account_info` lamports are already checked to be greater than or equal
// to the minimum balance.
account.set_amount(new_account_info.lamports() - minimum_balance);
```